src/DAMA/DoctrineTestBundle/Doctrine/DBAL/AbstractStaticConnection.php

```diff
@@ -50,6 +50,13 @@ public function rollBack(): bool
     }
 
     public function getWrappedConnection(): Connection
+    {
+        trigger_error('getWrappedConnection() is deprecated, call getNativeConnection() instead.', E_USER_DEPRECATED);
```

Comment:
actually this method is not deprecated. This is a method that I defined for accessing the wrapped/decorated driver connection.

The deprecated method is `Doctrine\DBAL\Connection::getWrappedConnection()`.

So need for this change.